### PR TITLE
[ENHANCEMENT] E2E Permission system tests

### PR DESCRIPTION
**Pull request by cortisiko**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

This PR adds e2e test coverage for the permission system. 

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Comment by cortisiko** [comment]
Browserstack runs for the tests: https://app-automate.browserstack.com/dashboard/v2/builds/8921ccc14ac9135ca38531b8fc30448c28a5e573/sessions/2bdb86881041963750f251b97e450e384c6a00cd 

and this: https://app-automate.browserstack.com/dashboard/v2/builds/8921ccc14ac9135ca38531b8fc30448c28a5e573/sessions/4425a4290f1ec7c39490765de1f7af18fed0a898

**Comment by cortisiko** [comment]
Detox is pending. Will leave a message with the build run later.

File: wdio/features/AppLaunchTime.feature [hunk 1]
```diff
@@ -7,7 +7,7 @@ Feature: App launch times
   Scenario: Measure app launch time on fresh install
     Given the app is launched
     When the app displayed the splash animation
-    Then the app should launch within x seconds
+    Then the app should launch within "5" seconds
```
Comment (Andepande) [inline wdio/features/AppLaunchTime.feature:10]: is 5 seconds now the benchmark?

**Review by Andepande — APPROVED** [review]
Are you able to provide screenshots of the test run and links to Browserstack runs?

**Review by SamuelSalas — APPROVED** [review]

File: wdio/features/AppLaunchTime.feature (hunk 1 above)
Comment (cortisiko) [reply wdio/features/AppLaunchTime.feature]: actually it is 4 seconds. This was handled in a different [PR](https://github.com/MetaMask/metamask-mobile/pull/5947).

**Review by giolicer — APPROVED** [review]

**Pull request merged**